packages/pwa-kit-react-sdk/src/config/schemas/v1-0-0.json

```diff
@@ -0,0 +1,264 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://example.com/object1636505239.json",
+    "title": "Root",
+    "type": "object",
+    "required": ["version", "app", "server"],
+    "properties": {
+        "version": {
+            "$id": "#root/version",
+            "title": "Version",
+            "type": "string",
+            "default": "",
+            "examples": ["1.0.0"],
+            "pattern": "^.*$"
+        },
+        "app": {
```

Comment:
It does not really make sense have this `app` schema in the SDK. If we take a closer look into it, why does it have to be aware of commerceApi or einstainApi or anthing `app-related` config at all? 
Image if we were to use SDK to build another pwa that does not use any of these commerce api, how does the app schema fit into that? 

The only section that makes sense to the SDK is the `server`

is it possibe for to have two schemas, one for the SDK and one for the app specific?

Reply:
I kind of agree with Alex. I think we would need to take a deeper look at what sort of config this schema + config file contains once we consolidate them. I am also thinking if we would extract some of the SFCC specific configurations into another module and make it plug and play with the effort for hooks extraction. Looking at @bendvc for thoughts there.

Reply:
> is it possibe for to have two schemas, one for the SDK and one for the app specific?

The schemas are for validation purpose. So is the suggestion here to have the app validate itself?

I feel like the SDK should know more about how to properly validate the configs, whether they're for server or the app.

Reply:
How about making the `commerceApi` and `einsteinApi` optional? We don't assume they will be there on every PWA. But if they are, then we'll validate their configs.